common/logger/src/diem_logger.rs

```diff
@@ -128,6 +128,7 @@ pub struct DiemLoggerBuilder {
     address: Option<String>,
     printer: Option<Box<dyn Writer>>,
     is_async: bool,
+    custom_format: Option<Box<dyn Fn(&LogEntry) -> Result<String, fmt::Error> + Send + Sync>>,
```

Comment:
Curious, what's the custom format that you're looking for that's not in there now?

Reply:
In AOS we have quite a few dependencies that use the `log` crate therefore our logs are currently a mix of the default log format for `env_logger` and the default format for `diem_logger`. One solution was to change the env_logger format to match the diem_logger format, but seems like having the option of changing the default format would be a nice feature, thinking about the case where we'd want the whole log output to be json for instance, including timestamp and all